src/components/filters/HourRange.tsx

```diff
@@ -29,7 +30,9 @@ function HourlyRangeFilter({ range, setRange }: Props) {
 
     return (
         <Stack direction="row" spacing={1} sx={{ alignItems: 'center' }}>
-            <FormattedMessage id="detailsPanel.recentUsage.title.prefix" />
+            <Typography id="hourly-filter-selector__label">
```

Comment:
The styling of all card headers should be uniform. This header is missing the following: `{ fontSize: 16, fontWeight: 300 }`.

It should be noted that the heading of the data preview card is similarly divergent.